README.md

```diff
@@ -161,13 +161,23 @@ Golang import:
 import "gopkg.in/confluentinc/confluent-kafka-go.v1/kafka"
 ```
 
-**Note:** that the development of librdkafka and the Go client are kept in synch.
-If you use the master branch of the Go client, then you need to use the master branch of
-librdkafka.
+librdkafka is included with the Go client and does not have to be installed
+separately on the build, nor target, system.
+
+
+**Dynamic linking:** If you prefer to link librdkafka dynamically, which requires
```

Comment:
maybe: "If you prefer to link librdkafka dynamically you should build your application with `go build -tags dynamic`. This requires librdkafka to be installed on your system.  ..."